### ISSUE TITLE
Add ability to cutomize blackfire based images

### ISSUE THREAD

**Issue by ihor-sviziev** [wardenenv/warden#285]
Hi,

I started using warden on our own project. I need to use custom php-fpm and php-debug images because we're using additional PHP extensions that are not bundled with Warden (in the `.warden/warden-env.yml` file).
So far, I wasn't able to find a way to override the php-blackfire image.

**Comment by navarr** [comment]
Can you confirm how you attempted to override them?

I would think they'd be overrideable by specifying a different image for the `php-blackfire` service, and the `blackfire-agent` service if necessary

**Comment by ihor-sviziev** [comment]
Basically, if I'll add them to the `.warden/warden-env.yml` file - these services will also be created even when I'll have `WARDEN_BLACKFIRE=0` in my .env config

**Comment by navarr** [comment]
I don't have time to attempt this right now, so I'm going to document my notes here:

This might be possible by updating https://github.com/davidalger/warden/blob/495176185a6cfbdff5f775b3f91ecc84d7dd3959/utils/env.sh#L77 such that it also searches the WARDEN_ENV_PATH for matching files.

**Comment by davidalger** [comment]
Apart from changing lines like this:
```
    image: docker.io/wardenenv/php-fpm:${PHP_VERSION:-7.4}${WARDEN_SVC_PHP_VARIANT:-}-blackfire
```
to something like:
```
    image: ${WARDEN_SVC_PHP_BLACKFIRE_IMAGE:-docker.io/wardenenv/php-fpm}:${PHP_VERSION:-7.4}${WARDEN_SVC_PHP_VARIANT:-}-blackfire
```
so the image name could be overridden (would still require the same tagging format), I think the only way to make it happen is what @navarr suggests adding `WARDEN_ENV_PATH` to the list of search paths `appendEnvPartialIfExists` is using. 

If someone wants to put together a PR for this, I'd accept it. Curious edge case, but common enough I'm wondering if adding variables to allow easily replacing the image might be a better solve.

**Comment by Den4ik** [comment]
I think it would be good to provide the ability to override not only Blackfire images but all images used in projects

**Comment by ihor-sviziev** [comment]
Let's keep this issue open

**Comment by ihor-sviziev** [comment]
Still actual

**Comment by ihor-sviziev** [comment]
Keep it open

**Comment by Den4ik** [comment]
Actual

**Comment by navarr** [comment]
I believe this is resolved by 5a24ca7d50790e35d7856ef5aaa9002040ff5e46.  Please correct me if I am wrong.
